mix.exs

```diff
@@ -8,7 +8,15 @@ defmodule GoldenPath.MixProject do
       elixir: "~> 1.5",
       elixirc_paths: elixirc_paths(Mix.env()),
       compilers: [:phoenix, :gettext] ++ Mix.compilers(),
+      dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:mix]],
```

Comment:
Configure Dialyzer to use Persistent Lookup Table (PLT) for faster static analysis, tell it where to find our mix tasks so it doesn't error out, and recursively add dependencies as well.